deployVersion.sh

```diff
@@ -61,13 +61,13 @@ npm install
 
 # Pretranslate all the CSS
 echo "Building app."
-./gulp sass
+./node_modules/.bin/gulp sass
```

Comment:
Lando is currently set up to use the former symlink, but it can be changed to use the full path to gulp

Reply:
Yes, please point to the file in node_modules. This change is needed because of the order in which hooks are executed and unzip is performed. I moved these links to nodein.sh

Reply:
I'm not currently able to get a Lando development environment working, with these changes added. I will keep trying tomorrow, to see if I can make any suggestions.